coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -15,15 +15,31 @@
         v-for="(item, index) in orderBy(taskItems, 'order')"
         v-bind:key="index"
         expand-icon=""
-        :class="{ 'disabled': !item.enabled, 'draft': isDraft(item) }">
+        :class="{ 'disabled': !item.enabled, 'draft': isDraft(item),
```

Comment:
Should these conditions be mutually exclusive -- what happens if all 3 classes are applied at once? Ref: https://vuejs.org/v2/guide/class-and-style.html

Reply:
Good question, i do not have the answer :) I will look into this!

Reply:
In this instance, if all three classes are applied at once the expansion panel would be `display: none` as it would supersede the other styles. That being said, the classes 'draft' and 'awaitConfirm' couldn't both exist on the panel due to logic that only enables the button on a BCOMPs New Annual Reports ( whom have no drafts ). The 'disabled' class can exist under many situations tho, so i would say that these three classes are not mutually exclusive. That is an interesting read though, and i dig into it a little deeper and see if there is more improvements to be made. I'm sure there is so much more we can do with Vue still yet :)

Reply:
Thanks for looking into this.